libyul/optimiser/StackCompressor.cpp

```diff
@@ -168,7 +168,7 @@ bool StackCompressor::run(
 	bool allowMSizeOptimzation = !MSizeFinder::containsMSize(_dialect, *_object.code);
 	for (size_t iterations = 0; iterations < _maxIterations; iterations++)
 	{
-		map<YulString, int> stackSurplus = CompilabilityChecker::run(_dialect, _object, _optimizeStackAllocation);
+		auto stackSurplus = CompilabilityChecker(_dialect, _object, _optimizeStackAllocation).stackDeficit;
```

Comment:
Hm... I'm only seeing this now... which one should I rename :-D? Or is it fine :-)?

Reply:
names are fine. what is not fine is the auto.

Reply:
@ekpyron so apparently `auto` is still a complaint here?

Reply:
No, it's not there anymore since last month :-).